jetty-openid/src/main/config/etc/jetty-openid.xml

```diff
@@ -1,12 +1,24 @@
 <?xml version="1.0"?>
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Get id="ThreadPool" name="threadPool"/>
+  <New id="HttpClient" class="org.eclipse.jetty.client.HttpClient">
+    <Arg>
+      <New class="org.eclipse.jetty.util.ssl.SslContextFactory$Client">
+        <Set name="trustAll" type="boolean">false</Set>
```

Comment:
Why not use a property to set this? Setting this to true is very common in debugging and testing situations, and it would be nice to not have to "own" all this XML but to simply update openid.mod with a flag. Hightly recommend changing back to `<Property name="jetty.openid.trustAllCertificates" default="false"/>`

Reply:
I agree this should probably be a property.... but let's not make too many more things here properties as the main method for any significant configuration will still be writing new XML for this one.